scst-store/using-encryption-and-connection.hbs.md

```diff
@@ -20,39 +20,9 @@ For a production environment, VMware recommends that the Store is installed with
 
 ## <a id="ingress"></a>Using `Ingress`
 
-When using an [Ingress setup](ingress-multicluster.md), the Store creates a 
-specific TLS Certificate for HTTPS communications under the `metadata-store` namespace.
+When using an [Ingress setup](ingress-multicluster.md), the Store creates a specific TLS Certificate for HTTPS communications under the `metadata-store` namespace.
 
-To get a certificate, run:
-
-```bash
-kubectl get secret ingress-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d > insight-ca.crt
-```
-
-The endpoint host is set to `metadata-store.<ingress-domain>`, 
-such as `metadata-store.example.domain.com`). 
-
-This value matches the value of `ingress_domain`.
-
-If no accessible DNS record exists for the domain, edit the `/etc/hosts` file to add a local record:
-
-```bash
-ENVOY_IP=$(kubectl get svc envoy -n tanzu-system-ingress -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
-
-# Replace with your domain
-METADATA_STORE_DOMAIN="metadata-store.example.domain.com"
-
-# Delete any previously added entry
-sudo sed -i '' "/$METADATA_STORE_DOMAIN/d" /etc/hosts
-
-echo "$ENVOY_IP $METADATA_STORE_DOMAIN" | sudo tee -a /etc/hosts > /dev/null
-```
-
-Set the target by running:
-
-```bash
-tanzu insight config set-target https://$METADATA_STORE_DOMAIN --ca-cert insight-ca.crt
-```
+{{> 'partials/insight-ingress-configuration' }}
```

Comment:
Sorry, this isn't the right place for this comment but I'm not sure where else to put this. 

I think there could be more done with partials in this file to make it easier to follow top-down.

Reply:
yep there are more changes to make this file better, just haven't been pushed up yet.